src/interfaces/correction-filing-interface.ts

```diff
@@ -21,7 +23,7 @@ export interface CorrectionFilingIF {
   correction: {
     correctedFilingId: string
     correctedFilingType: string
-    correctedFilingDate: string
+    correctedFilingDate: IsoDatePacific
```

Comment:
IsoDatePacific  is an alias to a string, but the name gives us extra meaning.

In shared repo:
```
/**
 * ISO date string (YYYY-MM-DD), assumed to be in Pacific timezone.
 * @example "2021-12-31"
 */
export type IsoDatePacific = string
```